src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -84,6 +84,7 @@ class Connection
      *     * exchange_name: Name of the exchange to be used for the retried messages (Default: "retry")
      *   * auto-setup: Enable or not the auto-setup of queues and exchanges (Default: true)
```

Comment:
looks like this should be `auto_setup`? @weaverryan?

Reply:
On the list - but if someone else wants to change this... that would be awesome :)

Reply:
I will look into it :+1:

Reply:
See https://github.com/symfony/symfony/pull/30702

Reply:
♥️♥️